pom.xml

```diff
@@ -79,6 +79,12 @@
       <version>2.10.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Can we pick one of these or the other?  There is already a ton of hamcrest usage everywhere. I'm not sure that there is enough value in adopting another library that people must then learn to understand our tests.

Reply:
We are slowly migrating to a better world ;-)

Reply:
I've removed it. And have updated a bunch of dependencies. Plus replaced findbugs by spotbugs.